tests/render_test.py

```diff
@@ -33,3 +36,54 @@ def test_join_to_basic():
     ]
     assert join_to(foreign_keys, table) == [{'id': '1', 'data': 'a'}, None]
     assert join_to(foreign_keys, table, 'data') == [None, None]
+
+
+class TestRendering:
+    def setup(self):
```

Comment:
I use a pytest fixture for something similar (but not quite the same) here:

https://github.com/innolitics/rdm/blob/master/tests/install_hooks_test.py#L9

I don't have a strong opinion, but I do tend to think pytest fixtures are easier to read and reason about.  Also, since we are using fixtures elsewhere, it could be nice to stick to one or the other (I think pytest fixtures can do everything that `setup` and `teardown` can do, but I may be mistaken).

Reply:
It took some finesse to get Travis-Ci to work, so I am skipping your suggestion for now.
It might be worth revisiting and/or discussing at some point.

Reply:
Good call, thanks!